code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -705,6 +705,12 @@
 				to_chat(src, "<span class='notice'>You succesfuly remove the durathread strand.</span>")
 				remove_status_effect(STATUS_EFFECT_CHOKINGSTRAND)
 			return
+		else if(islizard(src) && creamed)
```

Comment:
It shouldn't check if you're a lizard.
It shouldn't instead check if you have a lizard tongue because a lizard can not have a tongue or someone else can have a lizard tongue.

Reply:
Done